### PR TITLE
rename `application` to `app`

### PR DESCRIPTION
**Pull request by totoprayogo1916**
Just rename folder name in UG 
`application` to `app`.

**Checklist:**
- [x] Securely signed commits
- [x] User guide updated

### CODE REVIEW

**Review by paulbalandan — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "phpstan/phpstan": "0.12.88",
         "phpunit/phpunit": "^9.1",
         "predis/predis": "^1.1",
-        "rector/rector": "0.11.16",
+        "rector/rector": "0.11.20",
```
Comment (MGatner) [inline composer.json:26]: I'm not sure how this got in here, maybe you are working off an old branch? Please rebase and force push.

**Review by MGatner — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (totoprayogo1916) [reply composer.json]: @MGatner : its (#4832) already merged  to develop branch

Comment (totoprayogo1916) [reply composer.json]: and now, I have deleted the previous commit

Comment (MGatner) [reply composer.json]: Thanks!

**Review by MGatner — APPROVED** [review]

**Pull request merged**